plugins/env/app/assets/javascripts/env/application.js

```diff
@@ -49,14 +49,39 @@
     return $new_row;
   }
 
+  // Update the query string of the associated preview link
+  function formatPreview($preview, val) {
+    if (val) {
+      $preview[0].search = $.param({ group_id: val });
+      $preview.show();
+    } else {
+      $preview.hide();
+    }
+  }
+
+  function setupPreview($input) {
```

Comment:
add a comment explaining what it does and pick a less generic name `setupEnvGroupPreview`

Reply:
like "When the user does A we do B"